### PR TITLE
Stacktester updates

### PR DESCRIPTION
**Pull request by dwalleck**
Added file injection to server build test, added ssh verification to rebuild resize tests, removed deprecated known issue tags

### CODE REVIEW

File: stacktester/tests/test_server_actions.py [hunk 1]
```diff
@@ -120,6 +120,15 @@ class ServerActionsTest(unittest.TestCase):
...
         self.assertTrue(new_pwd_ssh_client.test_connection_auth())
+        
+        #Revert password change
+        post_body = json.dumps({
+            'changePassword' : {
+                'adminPass' : 'testpwd'
+            }
+        })
+        url = '/servers/%s/action' % self.server_id
+        response, body = self.os.nova.request('POST', url, body=post_body)
```
Comment (throughnothing) [inline stacktester/tests/test_server_actions.py:131]: I see what you're doing here, should we maybe assert once again that the password has successfully changed back?  That way if that fails, we'll know here right away?

**Comment by throughnothing** [comment]
This looks pretty good to me, other than my one comment.

**Comment by bcwaldon** [comment]
Agree w/ Will. Address that one comment then this can go in.

**Comment by dwalleck** [comment]
Okay, added validation for the password after reverting

**Comment by throughnothing** [comment]
Looks good to me, thanks Daryl.

**Pull request merged**